File: moban_java/src/main/java/org/jdk/project/dto/config/AudioUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class AudioUpsertRequest {
  private Long loungeId;
  private String audioKey;
  private String audioType;
  private String languageType;
  private String textInfo;
  private String audioValue;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/AudioUpsertRequest.kt
package org.jdk.project.dto.config

data class AudioUpsertRequest(
    var loungeId: Long? = null,
    var audioKey: String? = null,
    var audioType: String? = null,
    var languageType: String? = null,
    var textInfo: String? = null,
    var audioValue: String? = null,
    var remark: String? = null,
)
